library/proptest/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+[package]
```

Comment:
I'm not sure if this is the right place for it, but you should discuss somewhere the overriding strategy.

Reply:
The overriding "idea" doc is added to Developer Guide marked as experimental.  https://github.com/model-checking/kani/pull/1435/commits/1a549a4859ce6fa5bf53b5ea81c3e36a9a655b72